bgpd/bgp_nht.c

```diff
@@ -704,6 +704,8 @@ static void bgp_nht_ifp_table_handle(struct bgp *bgp,
 {
 	struct bgp_nexthop_cache *bnc;
 
+	assert(ifp->ifindex != IFINDEX_INTERNAL);
```

Comment:
Safe, but also a crash. Shouldn't we print a warning and just return here?

Reply:
Done, please help check it, thanks!